HogeFuga/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="hogefuga"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/7bc72839ceeb91362b29abe3d6e45fdb9be34890/HogeFuga/app/src/main/res/layout/activity_main.xml#L12"></span>Hardcoded string “hogefuga”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>